pkg/authorization/authorizer/authorizer.go

```diff
@@ -394,6 +436,26 @@ func (a *openshiftAuthorizationAttributeBuilder) GetAttributes(req *http.Request
 		return nil, errors.New("could not get user")
 	}
 
+	isNonResourceURL := a.infoResolver.APIPrefixes.Has(req.URL.Path[1:])
```

Comment:
Can you invert the naming here?  I'm having trouble scanning this.  It *looks* wrong as is